iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AttachLoad.java

```diff
@@ -0,0 +1,265 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification.topology;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.reporter.Report;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.TypedValue;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.modification.NetworkModification;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.extensions.BusbarSectionPosition;
+import com.powsybl.iidm.network.extensions.ConnectablePosition;
+import com.powsybl.iidm.network.extensions.ConnectablePositionAdder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.stream.Stream;
+
+import static com.powsybl.iidm.modification.ModificationUtils.throwExceptionOrLogError;
+import static com.powsybl.iidm.modification.topology.TopologyModificationUtils.*;
+
+/**
+ * This method adds a new load on an existing voltage level. The voltage level should be described
+ * in node/breaker topology.
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class AttachLoad implements NetworkModification {
```

Comment:
To rename: AttachLoad does not mean anything

Reply:
It was named like that mainly because of the existing `AttachVoltageLevelOnLine` and `AttachNewLineOnLine`, but you suggested in #2218 to rename them by replacing `Attach` into `Insert`. Shouldn't we use the same prefix here and for those two other modifications? 

What was objected against the `Create` you suggested previously is that the `InjectionAdder` is given here, so that the load is already half-way to its creation. Isn't it more a connect (or insert, or ... attach!) operation then?

Reply:
No, until add has been called nothing has been creates, just some data have been filled

Reply:
A adder since add is not called is just a form, nothing else. So this this the modify method which call the add that create the injection and create also other topology stuffs.

Reply:
Regarding insert or create. In the the AttachVoltageLevelOnLine and AttachNewLineOnLine it is a creation, but more precisely we insert it into an existing equipment.
In this PR, it is really a creation PLUS a cell creation (all switches, internal connections).

Reply:
After checking for the right wording, we came to the conclusion that CreateBay is the best. The load is indeed created and connected with this modification. According to IEC60050, the correct word is Bay.

Reply:
Not sure, but maybe CreateFeederBay ? To be a little more specific. For example maybe we will want to have a CreateBusBarCouplerBay which will be different.